### PR TITLE
Added sfdx authURL authentication to smoke test pipeline

### PR DESCRIPTION
**Pull request by henry88lay**
Smoke tests have been repeatedly failing when pull requests are raised.  
Screen Shot 2021-08-16 at 4 33 37 pm

I looked into azure devOps to inspect the error and found this.
Screen Shot 2021-08-16 at 4 33 54 pm

Updated the authentication to AuthURL

### CODE REVIEW

File: templates/smoke-test-template.yml [hunk 1]
```diff
@@ -36,11 +36,12 @@ steps:
       script: |
         sfdx plugins:link
 
-  - task: sfpwowerscript-authenticateorg-task@9
-    displayName: "Authenticate  HubOrg using ServiceConnection"
+  - task: sfppowerkit-authenticateorg-task@9
```
Comment (azlam-abdulsalam) [inline templates/smoke-test-template.yml:39]: This should be a command line task

**Review by azlam-abdulsalam — CHANGES_REQUESTED** [review]
Please make the changes

**Comment by henry88lay** [comment]
authfile JSON has been uploaded to library.

**Comment by henry88lay** [comment]
Tested Pipeline and here are screenshots
Screen Shot 2021-08-17 at 1 32 37 pm

**Pull request merged**